### ISSUE TITLE
Some plans to "Git advanced GUI"?

### ISSUE THREAD

**Issue by MichelDiz** [keybase/client#9051]
I was wondering here. The Keybase client only maintains the repositories. Right? Is it possible in the future to have a GUI similar to Github?

To have functions like "talk about the code" with the team. Analyze "pull request" accept "PR" and delete "Branch" in the act.

If these things already have. Where do I find out about?

Cheers

**Comment by dabura667** [comment]
Yeah, I think this would be great... but the encrypted nature makes it very difficult to do something like fork and merge from forks.

It would almost necessitate the branch and merge method (rather than fork and merge) which would then almost definitely require the need for branch permissions...

It's a bit of a rabbit hole... but I think it should be on the roadmap.

But they are doing an AMAZING JOB. And I can't wait til they let me pay them.

**Comment by MichelDiz** [comment]
Yeah, that's an amazing tool.

At first I had not understood the potential of the keybase. After searching for videos and reading reviews I completely understood.

And I understand that the Git option is entirely new. I suppose my question was in their sights. I just wrote to probe it.

I do not know if it would be complicated. But I believe doing what I said on the client does not seem to be complicated like that. Because the client already opens with the user key, seems perfectly possible. In other words, it is authorized.

And overall everyone on the team shares access to the repository. So permissions would already be planned accordingly.

**Comment by expenses** [comment]
You're probably better off using a git gui on your end

**Comment by MichelDiz** [comment]
@expenses I think you didn't understood the point.

What I'm relating is the functions of a service such as "Github" or "Bitbucket" embedded in the Keybase Client.

Currently Keybase only hosts your files with Git. It would also be nice to be able to integrate this Git service with continuous integration. But for this I would have to add a seamless integration with the "Teams" logic that Keybase has. Authorizing the integration to my service. And this third continuous integration service would have to have a placeholder for integration with Keybase Git. It is a two-sided process, but Keybase would have to give the startup (first step) offering means of integration.
